### DISCUSSION TITLE
pyGeo In Windows?

### DISCUSSION THREAD

**Discussion by josedlp8** [mdolab/pygeo#244] [Q&A]
Hello all,

I am fairly new to the mdolab, but have been using OpenMDAO for some time. I am exploring the possibility of utilizing pyGeo with MPhys to do some MDO work with my team. However, it seems that pyGeo uses Linux. I am trying to see if it is possible of using Windows to run pyGeo? I would appreciate any help or guidance to doing so if possible.

**Comment by hajdik** [comment]
pyGeo cannot run natively on Windows due to dependencies written in Fortran. To run on Windows, you could try WSL or our [docker images](https://mdolab-mach-aero.readthedocs-hosted.com/en/latest/installInstructions/dockerInstructions.html).
